src/server/src/routes/index.ts

```diff
@@ -10,6 +11,7 @@ const router = Router();
 router.use('/sample', sampleRouter);
 router.use('/job', jobRouter);
 router.use('/facultyMemberProfile', facultyMemberRouter);
+router.use('/studentProfile', studentRouter);
```

Comment:
Now that I think about it, I think these routes should be named /student instead of /studentProfile. Same thing for /faultyMemberProfile -> /facultyMember

Reply:
Done.